src/taxonomy/data/api.js

```diff
@@ -18,6 +18,10 @@ export const getExportTaxonomyApiUrl = (pk, format) => new URL(
   `api/content_tagging/v1/taxonomies/${pk}/export/?output_format=${format}&download=1`,
   getApiBaseUrl(),
 ).href;
+export const getTaxonomyTemplateApiUrl = (format) => new URL(
+  `api/content_tagging/v1/taxonomies/import/template.${format}`,
+  getApiBaseUrl(),
+).href;
```

Comment:
This is just a general though, but I like using the URL object to manipulate URLs, but perhaps these functions can just return the object itself instead of `.href`? 
This is nothing wrong with this PR in particular, but that approach will then allow the API code to mainpulate those URL objects with ease, like adding parameters etc.

Reply:
I did that way to make it consistent with the other functions in the API. Do you mind if we leave this way?

Reply:
Definitely.